packages/forms/src/form_builder.ts

```diff
@@ -189,7 +245,17 @@ export class FormBuilder {
       formState: T|FormControlState<T>,
       validatorOrOpts?: ValidatorFn|ValidatorFn[]|FormControlOptions|null,
       asyncValidator?: AsyncValidatorFn|AsyncValidatorFn[]|null): FormControl {
-    return new FormControl(formState, validatorOrOpts, asyncValidator);
+    let newOptions: FormControlOptions = {};
+    if (!this.useNonNullable) return new FormControl(formState, validatorOrOpts, asyncValidator);
```

Comment:
No single line `if` statements / statements without braces:
```suggestion
    if (!this.useNonNullable) {
      return new FormControl(formState, validatorOrOpts, asyncValidator);
    }
```